src/components/TopSection.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import '../styles/components/App.css';
+import GooglePlay from '../svg/google-play-badge.svg';
+import Appstore from '../svg/App_Store_Badge.svg';
+import AvailableHotels from './AvailableHotels';
+import Logo from '../svg/logo_vector.svg';
+import Night from '../svg/Night.svg';
+import Account from '../svg/AccountCircle.svg';
+import MenuBurger from '../svg/menu.svg';
+
+class TopSection extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      text: '',
+      items: [],
+      searchSubmitHandler: this.searchSubmit.bind(this),
+      searchFileHandler: this.searchChange.bind(this),
+    };
+  }
+
+  fetchData(query) {
+    return fetch(`https://fe-student-api.herokuapp.com/api/hotels/popular?${new URLSearchParams({
+      search: query,
+    })}`).then((x) => x.json());
+  }
+
+  searchChange(e) {
+    e.preventDefault();
+    const newText = e.target.value;
```

Comment:
and here you can use  this.setState({ text: this.textRef.current.value }); here in place of event.target.value